packages/components/src/core/Tooltip/__tests__/__snapshots__/index.test.tsx.snap

```diff
@@ -9,18 +9,33 @@ exports[`<Tooltip /> Default story renders snapshot 1`] = `
   <div
     style="margin: 135px 300px;"
   >
-    <svg
-      aria-hidden="true"
-      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
+    <button
+      aria-label="tooltip test button"
+      class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium  css-1ha43wd-MuiButtonBase-root-MuiIconButton-root"
       data-mui-internal-clone-element="true"
-      data-testid="InfoOutlinedIcon"
-      focusable="false"
-      viewBox="0 0 24 24"
+      icon="[object Object]"
```

Comment:
just wanted to double check that we are expecting an object for `icon` 😁 Thank you!

Reply:
Eeew, it doesn't look good for sure :D 
I'll have to check it 😯

Reply:
Yup, you're right! there was a problem :D

Reply:
Amazing thanks for the fix 👏 !!